code/obj/item/device/pda2/modules.dm

```diff
@@ -311,6 +311,11 @@
 		var/area/A = get_area(src.host)
 		signal.data["message"]  = "<b><span class='alert'>***SECURITY BACKUP REQUESTED*** Location: [A ? A.name : "nowhere"]!"
 		src.host.post_signal(signal)
+		playsound(src, "sound/items/security_alert.ogg", 90, 1, 3)
+		var/map_text = null
+		map_text = make_chat_maptext(usr, "Emergency alert sent. Please assist this officer.", "color: #bbbbbb; font-size: 7px;", alpha = 180)
+		for (var/mob/O in hearers(usr))
```

Comment:
This is incredibly specific, but cogs likes to do fake sec alerts via proc calling and this will make people around your ghost or whatever see the alert which would be weird
would rather just add a `mob/user` argument to the `send_alert` proc, and replace all usage of `usr` in here with `user`. Then you'd call it in `Topic` with `send_alert(usr)`
basically that just lets you send alerts with a null user in case you want to do Ghost Alerts
also using `usr` in procs generally leads to weird bugs so w/e